docs/src/pages/components/text-fields/ColorTextFields.js

```diff
@@ -1,21 +1,17 @@
 import * as React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
 import TextField from '@material-ui/core/TextField';
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    '& > *': {
-      margin: theme.spacing(1),
-      width: '25ch',
-    },
-  },
-}));
-
 export default function ColorTextFields() {
-  const classes = useStyles();
-
   return (
-    <form className={classes.root} noValidate autoComplete="off">
+    <Box
+      component="form"
+      sx={{
+        '& > :not(style)': { m: 1 },
```

Comment:
Was the `width` intentionally not added?

Reply:
@mnajdova Yes, it had been intentional, but now checking in mobile I realize that the size is different ([comment](https://github.com/mui-org/material-ui/pull/25626#discussion_r608106416)). So I will add it again

Reply:
Resolved